com.microsoft.mrtk.spatialmanipulation/ObjectManipulator/ObjectManipulator.cs

```diff
@@ -614,6 +615,12 @@ public override void ProcessInteractable(XRInteractionUpdateOrder.UpdatePhase up
                             }
                         }
 
+                        // Immediately apply scale constraints after computing the user's desired scale input.
+                        if (EnableConstraints && constraintsManager != null)
```

Comment:
Should these `if`s be up in their corresponding `if (allowedManipulations.IsMaskSet(TransformFlags.*))` blocks? We were previously checking `allowedManipulations` for each of them in `ModifyTargetPose`